docs/how-to-guides/integrating-autoware/creating-vehicle-interface-package/vehicle-interface.md

```diff
@@ -0,0 +1,63 @@
+# Vehicle interface
+
+## What is the vehicle interface?
+
+The purpose of the vehicle interface package is to convert the control messages calculated from the autoware into a form that the vehicle can understand and transmit them to the vehicle (CAN, Serial message, etc.) and to decode the data coming from the vehicle and publish it through the ROS 2 topics that the autoware expects.
+So, we can say that the purposes of the vehicle_interface are:
+
+1. Converting Autoware control commands to a vehicle-specific format. Example control commands of autoware:
+2. Converting vehicle status information in a vehicle-specific format to Autoware messages.
+
+Autoware publishes control commands such as:
+
+- Velocity control
+- Steering control
+- Car light commands
+- etc.
+
+Then, the vehicle interface converts these commands into actuation such like:
+
+- Motor and brake activation
+- Steering-wheel operation
+- Lighting control
+- etc.
+
+So think of the vehicle interface as a module that runs the vehicle's control device to realize the input commands provided by Autoware.
+
+<figure markdown>
+  ![vehicle_interface_IO](images/Vehicle-Interface-Bus-ODD-Architecture.drawio.svg){ align=center }
+  <figcaption>
+    An example of inputs and outputs for vehicle interface
+  </figcaption>
+</figure>
+
+There are two types of controlling your own vehicle:
```

Comment:
I found the explanation of the two types hard to follow. Could we set a call to get aligned on what you are trying to say here?